src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties' :
+
+        Object.assign(state, action.extraData);
+
+        break;
+      case 'removeProperties' :
+        for (const remove of action.keysToRemove) {
```

Comment:
It's better to name it `key` instead of `remove` because you iterate `KEYStoRemove`.